pkg/orchestration/wiring/asapkey/plugin.go

```diff
@@ -43,11 +46,22 @@ func New() *WiringPlugin {
 			InstanceSpec:                  instanceSpec,
 			ObjectMeta:                    objectMeta,
 			ResourceType:                  ResourceType,
-			OptionalShapes:                svccatentangler.NoOptionalShapes,
+			OptionalShapes:                shapes,
 		},
 	}
 }
 
+func shapes(resource *orch_v1.StateResource, smithResource *smith_v1.Resource, context *wiringplugin.WiringContext) ([]wiringplugin.Shape, error) {
+	bindableEnvVarShape := knownshapes.NewBindableEnvironmentVariables(smithResource.Name, ResourcePrefix, map[string]string{
+		"PRIVATE_KEY": "data.private_key",
+		"ISSUER":      "data.issuer",
+		"KEY_ID":      "data.key_id",
+		"AUDIENCE":    "data.audience",
+	})
+	bindableEnvVarShape.Data.ExcludeResourceNameInKey = true
```

Comment:
Why not make this field a constructor parameter?

Reply:
The field is usually false, it seemed like a bit tedious to always provide false as an argument.

I don't have a strong opinion here so I can just add it into the constructor.